android/src/com/mapswithme/maps/base/BaseToolbarActivity.java

```diff
@@ -25,7 +25,8 @@ protected void onCreate(Bundle state)
     displayToolbarAsActionBar();
   }
 
-  protected @StringRes int getToolbarTitle()
```

Comment:
Ну я тоже сомневался долго, но в итоге решил что лучше делать как в самом SDK ставят. Мержи если LGTM)
> On Mar 15, 2016, at 3:28 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/base/BaseToolbarActivity.java <https://github.com/mapsme/omim/pull/2234#discussion_r56154265>:
> 
> > @@ -25,7 +25,8 @@ protected void onCreate(Bundle state)
> >      displayToolbarAsActionBar();
> >    }
> >  
> > -  protected @StringRes int getToolbarTitle()
> Но это так, придирка) LGTM
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/mapsme/omim/pull/2234/files#r56154265 <https://github.com/mapsme/omim/pull/2234/files#r56154265>